client/app/visualizations/table/utils.js

```diff
@@ -64,11 +64,16 @@ export function prepareColumns(columns, searchInput, orderBy, onOrderByChange) {
       align: column.alignContent,
       title: (
         <React.Fragment>
-          <Tooltip placement="top" title={column.title}>
-            <div className="table-visualization-heading" data-sort-column-index={sortColumnIndex}>
+          <div className="table-visualization-heading" data-sort-column-index={sortColumnIndex}>
+            {column.description && (
+              <Tooltip placement="top" title={column.description} className="p-r-5">
+                <i className="fa fa-info-circle" aria-hidden="true"></i>
+              </Tooltip>
+            )}
+            <Tooltip placement="top" title={column.title}>
```

Comment:
It breaks existing behavior for too long columns titles (before and after):
![image](https://user-images.githubusercontent.com/12139186/80227596-8b43b080-8656-11ea-8f55-41070def4cdb.png)
![image](https://user-images.githubusercontent.com/12139186/80227673-a57d8e80-8656-11ea-98bb-8af7dae08ef6.png)

Reply:
Thanks!

https://github.com/getredash/redash/pull/4831/commits/17432a2ec7be9ef576181ce59d41d090bb7160e6